packages/mocha/README.md

```diff
@@ -1 +1,102 @@
 # @bigtest/mocha
+
+Convergent Mocha functions for testing against asyncronous states
+
+## What is this package?
+
+This package simply wraps Mocha's `it` in a convergence helper so
+that any assertions you write using `it` become convergent assertions
+that allow you to easily test asyncronous states.
+
+### Convergent Assertions?
```

Comment:
I would lead with this. It explains the problem handily, and then we can talk about how it solves the problem. Also, remove the question mark.